### ISSUE TITLE
Pagination: active page/item disappears

### ISSUE THREAD

**Issue by Mirodil** [react-bootstrap/react-bootstrap#2270]
first active page disappears from result

```
<Pagination
                      items={4}
                      maxButtons={3}
                      activePage={1}/>
```

![Image of Yaktocat](https://i.imgur.com/mZvPtUI.png)

the same result if `boundaryLinks=false` or `ellipsis=false`

**Comment by TYL3RJC** [comment]
I had this issue but it seemed okay when I used:
`<Pagination
    items={25}
    ellipsis={false}
    maxButtons={3}
    activePage={1} />`

**Comment by Mirodil** [comment]
Yes, but if you have 4 items there is problem

**Comment by taion** [comment]
fixed by #2271
